doc/algorithm.qbk

```diff
@@ -150,6 +150,23 @@ Copy all elements that satisfy the element predicate from the start of the input
 [endsect:Copy]
 
 
+[section:Move Variations on Move]
```

Comment:
perform `chmod` on this file

Reply:
Fixed by `chmod 644 doc/algorithm.qbk && chmod 644 include/boost/algorithm/cxx11/move_if.hpp && chmod 644 test/Jamfile.v2 && chmod 644 test/move_if_test1.cpp`